### PR TITLE
[#175160851] Add possibility to update metadata for admin 

### PR DESCRIPTION
**Pull request by infantesimone**
To allow the metadata update for the admin user, the payload of the _PUT services_ is updated adding:
- `service_metadata : ServiceMetadata`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -77,9 +77,13 @@
     "shx": "^0.3.2",
     "ts-node": "^6.1.1",
     "tslint": "^5.8.0",
+    "io-functions-commons": "^14.6.0",
```
[the remark this answers is not in the document]
Comment (infantesimone) [reply package.json]: Done

**Comment by AleDore** [comment]
@gunzip : Can we merge this?

File: src/controllers/services.ts [hunk 2]
```diff
@@ -26,28 +21,11 @@ import {
...
 import { logger } from "../logger";
 
-import * as t from "io-ts";
-import { DepartmentName } from "../../generated/api/DepartmentName";
-import { MaxAllowedPaymentAmount } from "../../generated/api/MaxAllowedPaymentAmount";
-import { OrganizationName } from "../../generated/api/OrganizationName";
-
-import { ServiceName } from "../../generated/api/ServiceName";
-
-export const ServicePayload = t.partial({
```
Comment (gunzip) [inline src/controllers/services.ts:40]: it looks like all the parameter which where optional now are required, is this the expected outcome?

Comment (infantesimone) [reply src/controllers/services.ts:40]: It isn't the expected outcome, fixed

**Pull request merged**